### PR TITLE
download filename, error handling and cache busting

### PR DESCRIPTION
**Pull request by leangseu-edx**
Ticket
- [AU-482](https://openedx.atlassian.net/browse/AU-482)
- [AU-599](https://openedx.atlassian.net/browse/AU-599)

Screen Shot 2022-04-13 at 9 10 07 AM

### CODE REVIEW

File: src/data/services/lms/api.js [hunk 1]
```diff
@@ -52,7 +52,44 @@ const fetchSubmission = (submissionUUID) => get(
...
     [paramKeys.submissionUUID]: submissionUUID,
   }),
-).then(response => response.data);
+).then(response => ({
+  ...response.data,
+  response: {
+    ...response.data.response,
+    files: [
+      {
+        downloadUrl: 'samples/sample-1.mp4',
```
Comment (muselesscreator) [inline src/data/services/lms/api.js]: ?

Comment (leangseu-edx) [reply src/data/services/lms/api.js]: this is for sample test only. I will remove it from the final commit.

Comment (mattcarter) [reply src/data/services/lms/api.js]: Okay. If that's it and everything else looks good, let's pull it out

**Review by mattcarter — APPROVED** [review]
LGTM

**Review by muselesscreator — APPROVED** [review]
lgtm

**Pull request merged**